akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/ReceiveBuilder.scala

```diff
@@ -32,7 +32,7 @@ class ReceiveBuilder[T] private (
    * @param type type of message to match
    * @param handler action to apply if the type matches
    * @tparam M type of message to match
-   * @return a new behavior with the specified handling appended
+   * @return this behavior builder
    */
   def onMessage[M <: T](`type`: Class[M], handler: Function[M, Behavior[T]]): ReceiveBuilder[T] =
```

Comment:
should we align the naming?
In CommandHandlerBuilder it's `matchCommand`. Should this be `matchMessage`, or should it be `onCommand` in CommandHandlerBuilder?

Reply:
I think we should align. The only argument I can find either way for selecting which name is  that `onMessage` is shorter than `matchMessage`.

Reply:
perhaps also because pattern **match**ing is not a thing

Reply:
I prefer `on` then `match`. Just because it expresses better the fact that we are reacting to the arrival of something. 

The `match` variant aligns better to what happens behind, ie: we lookup for a handler that matches the criteria. We could even argue that `match` is leaking the implementation behind while `on` is descriptive to what should happen.

Making it short: +1 for `onMessage`, `onCommand` and `onEvent`.

Reply:
decided then!

Reply:
Created an issue to do that separately #26313